.kitchen.appveyor.yml

```diff
@@ -17,4 +21,11 @@ suites:
   - name: client
     run_list:
       - recipe[sql_server::default]
-    attributes: { sql_server: { accept_eula: true } }  
+    attributes: { sql_server: { accept_eula: true } }
+  # - name: install
```

Comment:
Can we just nuke this?

Reply:
yep I'll remove that the reboot doesn't appear to be working with appveyor.